flink-connector/flink/build.gradle.kts

```diff
@@ -37,15 +38,22 @@ val flinkMajorVersion: String = flinkVersion.substringBeforeLast(".")
 val scalaVersion: String = "2.12"
 val artifactName = "${rootProject.name}-flink-${flinkMajorVersion}_$scalaVersion"
 
+tasks.compileJava {
+  dependsOn(":catalogs:catalog-lakehouse-paimon:runtimeJars")
+}
+
 dependencies {
+  implementation(project(":core"))
   implementation(project(":catalogs:catalog-common"))
+
   implementation(libs.guava)
 
   compileOnly(project(":clients:client-java-runtime", configuration = "shadow"))
-
+  compileOnly(project(":catalogs:catalog-lakehouse-paimon"))
```

Comment:
It's odd to add dependencies to `:catalogs:catalog-lakehouse-paimon` for flink connector, it's a internal module of Gravitino server